test/e2e-tests.sh

```diff
@@ -121,10 +121,6 @@ toggle_feature tagHeaderBasedRouting Enabled config-network
 go_test_e2e -timeout=2m ./test/e2e/tagheader || failed=1
 toggle_feature tagHeaderBasedRouting Disabled config-network
 
-toggle_feature multi-container Enabled
-go_test_e2e -timeout=2m ./test/e2e/multicontainer || failed=1
-toggle_feature multi-container Disabled
```

Comment:
If we remove all then e2e test for multi container won't run :thinking: 

@dprotaso Shall we keep `toggle_feature` as it is in order to make sure even if disabled in `config-features` shouldn't effect the e2e ?